source/Configurations/Mock/Light

```diff
@@ -0,0 +1,31 @@
+{
+  "Mock": {
+    "Light": {
+      "module": "plugins.mock.light.Light",
```

Comment:
We can leave the collection name and plugin name because it is already due to the folder structure.